package.json

```diff
@@ -29,43 +29,47 @@
     "javascript"
   ],
   "dependencies": {
-    "chrome-launcher": "^0.13.4",
+    "chrome-launcher": "^0.15.2",
+    "d3": "5.16.0",
     "enquirer": "^2.3.5",
     "fs-extra": "^9.0.1",
     "get-port": "^5.1.1",
+    "js-levenshtein": "^1.1.6",
+    "lodash.clonedeep": "^4.5.0",
+    "lodash.throttle": "^4.1.1",
     "node-fetch": "^2.6.1",
     "open": "^7.2.1",
-    "puppeteer-core": "^15.4.0",
+    "prism-react-renderer": "^2.0.4",
+    "puppeteer-core": "^20.5.0",
+    "react": "16.13.1",
+    "react-dom": "16.13.1",
+    "react-flip-toolkit": "^7.0.12",
+    "react-is": "16.13.1",
     "request": "^2.88.2",
+    "sanitize.css": "11.0.1",
     "serve": "^11.3.2",
     "serve-handler": "^6.1.3",
     "source-map": "^0.7.3",
     "source-map-explorer": "^2.5.0",
     "static-server": "^2.2.1",
+    "styled-components": "^5.2.0",
     "tracium": "^0.2.1",
     "uuid": "^8.3.0",
     "yargs": "^16.0.3"
   },
   "peerDependencies": {
-    "puppeteer": "15.0.0"
+    "puppeteer": "^20.5.0"
   },
   "devDependencies": {
```

Comment:
the new dependencies such as `process` were added when running `yarn build` with the latest `parcel`